doc/nrf/releases_and_maturity/migration/migration_sysbuild.rst

```diff
@@ -570,19 +570,44 @@ They function the same in both systems:
 Building with sysbuild
 ======================
 
-With |NCS| 2.7, building with ``west`` will use sysbuild by default in |NCS| repositories.
-To maintain compatibility with child/parent images, out-of-tree applications will not use sysbuild by default.
-Twister does not inherit this change and will not use sysbuild unless tests are updated.
-Similarly, CMake will not configure projects using sysbuild unless the invocation command is updated.
+Sysbuild needs to be enabled from the command-line when building with ``west
+build``. This is achieved by passing the ``--sysbuild`` command-line parameter
+to it, or alternatively by configuring west itself to default to using sysbuild
+whenever invoking ``west build``. The latter can be achieved either
+per-workspace when using a local configuration option, or for all workspaces
+when using a global one:
+.. tabs::
 
-.. note::
+    .. group-tab:: local configuration
+
+        West will build all projects in the current workspace (including any
+        freestanding applications that are built against it) with sysbuild by default with the following command:
+
+        .. parsed-literal::
+           :class: highlight
 
-    Sysbuild can optionally be set as the default when using west.
-    This means that unless west is ran with ``--no-sysbuild`` then it will always use sysbuild
+           west config --local build.sysbuild True
 
-    .. code-block:: shell
+    .. group-tab:: global configuration
 
-        west config build.sysbuild True
+        West will build all projects in all workspaces with sysbuild by default  with the following command:
+
+        .. parsed-literal::
+           :class: highlight
+
+           west config --global build.sysbuild True
+
+No matter what is configured via ``west config``, the command-line parameters
+always take precedence. An additional parameter, ``--no-sysbuild`` exists as well
+to disable sysbuild when invoking ``west build``. In this manner, the usage of
+sysbuild can always be selected from the command-line, no matter how west has
+been configured.
+
+Additionally |NCS| 2.7 `modifies the default behavior <sdk-zephyr west build patch>`_ of ``west build``, so that building with ``west`` will use sysbuild by default in |NCS| repositories.
```

Comment:
```suggestion
Additionally |NCS| 2.7 modifies the default behavior (`sdk-zephyr west build patch`_) of ``west build``, so that building with ``west`` will use sysbuild by default in |NCS| repositories.
```

Reply:
or the link will not work

Reply:
Applied edits.